src/Enable.Extensions.Queuing.AzureServiceBus/Internal/AzureServiceBusQueueClient.cs

```diff
@@ -73,7 +73,7 @@ public class AzureServiceBusQueueClient : BaseQueueClient
             // This timeout is arbitrary. It is needed in order to return null
             // if no messages are queued, and must be long enough to allow time
             // for connection setup.
-            var message = await _messageReceiver.ReceiveAsync(TimeSpan.FromSeconds(10));
+            var message = await _messageReceiver.ReceiveAsync(TimeSpan.FromSeconds(5));
```

Comment:
This was necessary as 10 seconds was too long to return null if no messages were queued.
Based on a few tests, it was determined that 5 seconds consistently allows for null messages to be picked up.
After 7 seconds, it was determined that it fails to allow for null messages.

As stated in the comment, this is an arbitrary value and needs to be:
1. Short enough to allow nulls
2. Long enough to allow for connection setup.

All tests pass with this change. However, it's not possible to do local testing due to the need for Azure resources.
Therefore, I believe it's necessary to test the changes on Dev using the DealTrack API.